bridgesync/processor.go

```diff
@@ -575,6 +610,73 @@ func (p *processor) GetClaimsPaged(
 	return claims, claimsCount, nil
 }
 
+// GetUnsetClaimsPaged returns a paginated list of unset claims
+func (p *processor) GetUnsetClaimsPaged(
+	ctx context.Context, pageNumber, pageSize uint32,
+	networkIDs []uint32, globalIndex *big.Int,
+) ([]*UnsetClaim, int, error) {
+	whereClause := p.buildUnsetClaimsFilterClause(globalIndex)
+	claimsCount, err := p.GetTotalNumberOfRecords(ctx, unsetClaimTableName, whereClause)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	if claimsCount == 0 {
+		return []*UnsetClaim{}, 0, nil
+	}
+
+	offset, err := p.calculateOffset(pageNumber, pageSize, claimsCount, "unset claims")
+	if err != nil {
+		return nil, 0, err
+	}
+
+	orderByClause := orderByBlockDesc
+
+	rows, err := p.queryPaged(ctx, p.db, offset, pageSize, unsetClaimTableName, orderByClause, whereClause)
```

Comment:
We don't need the var here, WDYT?
```suggestion
	orderByClause := orderByBlockDesc

	rows, err := p.queryPaged(ctx, p.db, offset, pageSize, unsetClaimTableName, orderByBlockDesc, whereClause)
```